packages/documentation-framework/layouts/sideNavLayout/sideNavLayout.css

```diff
@@ -42,3 +42,9 @@
   --pf-v5-c-switch__input--not-checked__label--Color: var(--pf-v5-global--Color--100);
   --pf-v5-c-switch__input--checked__label--Color: var(--pf-v5-global--Color--100);
 }
+
+.ws-masthead .pf-v5-c-toggle-group {
+  --pf-v5-c-toggle-group__button--m-selected--BackgroundColor: var(--pf-v5-global--palette--blue-400);
+  --pf-v5-c-toggle-group__button--focus--BackgroundColor: transparent;
+  --pf-v5-c-toggle-group__button--hover--BackgroundColor: transparent;
+}
```

Comment:
Custom styling needed because otherwise the background colors on top of the dark masthead when in light mode looks off
![image](https://github.com/patternfly/patternfly-org/assets/7115053/a5fdb6c4-9179-46b3-8db6-d763be005d51)